pom.xml

```diff
@@ -44,7 +44,7 @@
 	</scm>
 
 	<properties>
-		<wicket.version>8.0.0-M3</wicket.version>
+		<wicket.version>8.0.0-SNAPSHOT</wicket.version>
```

Comment:
Thanks!
Can't wait M4 :))
Our release is blocked by this WebSocket/Session issue :(